tests/conftest.py

```
@@ -850,21 +844,14 @@ def mock_event():
 @pytest.fixture
 def test_config():
     """
-    Fixture providing a sample configuration for Meshtastic ↔ Matrix integration used by tests.
-
+    Provide a sample Meshtastic–Matrix integration configuration for tests.
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)